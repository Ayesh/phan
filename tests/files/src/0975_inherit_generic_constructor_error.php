<?php
/**
 * @template TModel
 */
class Foo extends Exception { }
$x = new Foo();
