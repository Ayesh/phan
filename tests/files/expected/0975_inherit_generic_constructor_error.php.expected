%s:5 PhanGenericConstructorTypes Missing template parameter for type TModel on constructor for generic class \Foo
